provisioning/buffer/kubernetes/templates/etcd.yaml

```diff
@@ -8,6 +8,17 @@ resources:
   limits:
     memory: "2000Mi"
 
+extraEnvVars:
+  # Set max DB size to 80GB
+  # 8GB is the recommended maximum, but we want to test how it works for our use-case.
+  # It's better to let it slow down than stop working at all.
+  - name: ETCD_QUOTA_BACKEND_BYTES
+    value: "85899345920"
+
+# Set disk size
+persistence:
+  size: 100Gi
+
```

Comment:
U etcd sa neodporuca mat vacsiu velkost DB ako `8GB`, aby korektne fungovali vsetky DB operacie.

Tusim, ze nam to nebude stacit.
Je sanca, ze nam vysia hodnota nebude vadiet, kedze v danom prefixe iba ukladame data a nactavame ich po strankach -tj. nerobi sa nic narocne na DB. 

S velkostou DB moze rast aj spotreba memory, ale opat tazko povedat ako velmi pri nasom use-case.

Preto by som to zdvihol, ... je lepsie, ked zistime, "tak nad 20GB" nam to ide uz dost pomaly, ako ked sa to pri 8GB zastavi pri chybe "no free space" a nezistime nic.

Zaujima bude vediet aj to, ci je problemom fyzicka velkost alebo logicka velkost.
Kedze data intenzivne mazeme, tak fyzicka velkost na disku pred defragmentaciou moze byt napr `50GB`, ale logicka velkost (tj. bez zmazanych dat) napr `1GB`.

Zvysena velkost db `8GB->80GB`, disk `10GB` -> `100GB`.

Reply:
Automaticky sa velkost disku nezmeni, na testingu to mozeme riesit vymazanim etcd stateful set a diskov.